### ISSUE TITLE
Make this project opensourced after close

### ISSUE THREAD

**Issue by FDiskas** [waffleio/waffle.io#3839]
Sad to but you are closing your project
Is it possible to publish the source code after the deadline?

So comunity could keep this nice project going on

**Comment by raiviszel** [comment]
So sad. Waffle board was one of the most used tools. Btw why are you closing it? Please share code it it is possible

**Comment by FDiskas** [comment]
this is good alternative https://www.gitkraken.com/glo

**Comment by jjlankinen** [comment]
I've been looking for replacement for Waffle Board but it seems to be impossible task as every other app just does not seem to offer enough features. I've been testing Zube, Zenhub, Github's Projects, Codetree, Pipefy, Fredcamp and many others and none of those does what we need so it's kind of sad situation.

Glo board started to look good but it lack multi-repo boards which is critical for us. It does not have very good automations either.

Hopefully you can release this version as open source :)

**Comment by FDiskas** [comment]
I also tried Glo and I managed to setup multirepo boards. Also looks like there is web hooks, and it synqs with github issues with no problem. What kind of automations you are talking about? :nine:

**Comment by jjlankinen** [comment]
How did you managed to show issues from multiple repos in one board in Glo board. I genuinely interested. I can't see any options for it.

Automation that I would like to have are quite simple. When new PR is made and in that PR is used github's closing keywords the issue and PR would be linked automatically and the issue would move to Need Review column etc. When merge happens I would like to move the related issue(s) to In Staging as we have automatic build process from our development branch to indicate that current feature is ready for testing in our staging server. Then when we do release I would like to move all related cards to In Production column. There are many other things that should be automated as we see it pretty redundant to manually move cards etc. when that time should be spent developing features and finxing bugs instead of moving cards around :)

**Comment by FDiskas** [comment]
Sorry about that big screenshot, but if I connect with github I can have multiple synced dashboards.
About thous moving tickets: I would go with smart commit messages https://help.github.com/en/articles/closing-issues-using-keywords
But this probably will not fit your needs. So it would make sense to use webhooks https://support.gitkraken.com/developers/webhooks/overview/

![Screenshot_20190327-231550](https://user-images.githubusercontent.com/468006/55112904-5d540b80-50e6-11e9-903d-6c3ce7fc183c.jpg)

**Comment by jjlankinen** [comment]
Yeah, I can do that too BUT repos don't appear on the same board. It does not do any good for me/our team as changing board all the time is very slow when you have sort of big backlog and it's not really desirable situation at all when our 2 or more projects are in different repos but are closely tied together so tasks needs to be viewed on the same board. More convenient this way. Think of it as backend and frontend being in different repo but they are dependant on each other in almost every issues.

Webhooks looked like they only work from glo to github. I need glo board to update automatically based on what hapens on github etc. and we already use closing keywords etc.

I just have to admit that there is no other as good as Waffle.io alternative for our workflow currently, been testing over 15 different boards and every one of them comes short in some way unfortunately.

This is a bit off topic but I think it shows that going open source would do a favor for at least some of us :)

Btw, thanks for trying to help out @FDiskas !

**Comment by nikku** [comment]
I've created my own open-source [task board](https://github.com/nikku/wuffle) in the spirit of Waffle. It may (or may not :wink:) evolve into a worthy waffle replacement some day in the future.

Thank you waffle guys for putting up such a helpful, good working tool :heart:.

**Comment by FDiskas** [comment]
Just like that? Couple days and its done?

**Comment by nikku** [comment]
> Just like that? Couple days and its done?

If you refer to my [previous comment](https://github.com/waffleio/waffle.io/issues/3839#issuecomment-484418409): Nope. Some more work and it is done.

**Comment by Joshuaalbert** [comment]
@FDiskas Let's stay on point in this issue.  Open source Waffle would be beyond wonderful. There are few tools like it, and it could have a positive impact on the open source community, e.g. on large science collaborations like the one I work in.

**Comment by meiqimichelle** [comment]
+100 -- our team hasn't been able to find a suitable replacement, either :-/ We also need to pull in many repos, across several orgs, and have a decent amount of automation. We're 😭 over here about Waffle going away 😢
